### PR TITLE
Update main.py to add kicks idk if it'll work lol

### PR DESCRIPTION
**Pull request by Zippybonzo**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -123,6 +123,11 @@ async def funfacts(interaction):
 async def warning(interaction: discord.Interaction, member: discord.Member, reason: str):
     await interaction.response.send_message(f" <a:warning:1126947077960765631> | Warned {member.mention} for {reason}, you {random.choice(insults)}")
 
+@tree.command(name="kick", description="Kick your friends and see if they rejoin!")
+async def warning(interaction: discord.Interaction, member: discord.Member, reason: str):
```
Comment (opened-src) [inline main.py:127]: Change:
```py
async def warning
```
to:
```py
async def kick
```

File: main.py [hunk 2]
```diff
@@ -123,6 +123,11 @@ async def funfacts(interaction):
...
+    del /guilds/{guild.id}/members/{user.id}
```
Comment (opened-src) [inline main.py:128]: According to the docs, use
```py
await member.kick(member=member)
```

**Review by bugsquashing — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (Zippybonzo) [reply main.py:128]: ok

**Comment by Zippybonzo** [comment]
how do you answer requested changes @opened-src

**Comment by Zippybonzo** [comment]
Lol @opened-src I think it's done....

File: main.py (hunk 1 above)
Comment (Zippybonzo) [reply main.py]: done

**Pull request closed without merging**